[2024-07-10 16:27:18,540] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: test_questions.bash2 manual__2024-07-10T16:27:16.481190+00:00 [queued]>
[2024-07-10 16:27:18,553] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: test_questions.bash2 manual__2024-07-10T16:27:16.481190+00:00 [queued]>
[2024-07-10 16:27:18,553] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-07-10 16:27:18,554] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2024-07-10 16:27:18,554] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-07-10 16:27:18,569] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): bash2> on 2024-07-10 16:27:16.481190+00:00
[2024-07-10 16:27:18,573] {standard_task_runner.py:52} INFO - Started process 3644 to run task
[2024-07-10 16:27:18,577] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'test_questions', 'bash2', 'manual__2024-07-10T16:27:16.481190+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/test_questions.py', '--cfg-path', '/tmp/tmpi5bgpggp', '--error-file', '/tmp/tmp9m1sgfb5']
[2024-07-10 16:27:18,577] {standard_task_runner.py:80} INFO - Job 3: Subtask bash2
[2024-07-10 16:27:18,698] {task_command.py:369} INFO - Running <TaskInstance: test_questions.bash2 manual__2024-07-10T16:27:16.481190+00:00 [running]> on host c8efd281e0f4
[2024-07-10 16:27:18,959] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_questions
AIRFLOW_CTX_TASK_ID=bash2
AIRFLOW_CTX_EXECUTION_DATE=2024-07-10T16:27:16.481190+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-07-10T16:27:16.481190+00:00
[2024-07-10 16:27:18,961] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2024-07-10 16:27:18,962] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls']
[2024-07-10 16:27:18,972] {subprocess.py:85} INFO - Output:
[2024-07-10 16:27:18,976] {subprocess.py:96} INFO - Command exited with return code 0
[2024-07-10 16:27:19,011] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=test_questions, task_id=bash2, execution_date=20240710T162716, start_date=20240710T162718, end_date=20240710T162719
[2024-07-10 16:27:19,042] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-07-10 16:27:19,113] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
